api/src/app/modules/app-config/configuration.ts

```diff
@@ -32,4 +36,10 @@ export default registerAs('app', () => ({
   port: process.env.APP_PORT,
   globalPrefix: process.env.APP_PREFIX,
   title: process.env.APP_TITLE,
-}));
+
+  // Forest Client APP Integration
+  fcApiBaseUrl: process.env.CLIENT_API_BASE_URL,
+  fcApiApiToken: process.env.CLIENT_API_TOKEN,
+  fcApiReqTimeOut: process.env.CLIENT_API_REQ_TIMEOUT,
+  fcApiBatchSerchPageSize: process.env.CLIENT_API_BTH_PAGE_SIZE,
```

Comment:
Spelling mistake -fcApiBatchSerchPageSize 'Serch' -> 'Search'.

Reply:
Thank you for finding it... I should double check again next time... This is fixed.